jqqb/rule.py

```diff
@@ -86,7 +115,7 @@ def typecast_value(self, value_to_cast):
             )
         elif self.type == "date":
             return (
-                datetime.strptime(value_to_cast, "%Y-%m-%d")
```

Comment:
Because the data is saved in this format %d/%m/%Y

```
File "/usr/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '01/07/2020' does not match format '%Y-%m-%d'
```

Reply:
Comme vu ensemble, on conserve ce standard et on corrigera plutôt le soucis dans les données générées par les fixtures.